aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -465,12 +466,17 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
 
     private fun handleBackspaceAndEnter(event: KeyEvent): Boolean {
         if (event.action == KeyEvent.ACTION_DOWN && event.keyCode == KeyEvent.KEYCODE_ENTER) {
-            return onKeyListener?.onEnterKey() ?: false
+            // Check if the external lister has consumed the enter pressed event
+            // In that case stop the execution
+            if (onAztecKeyListener?.onEnterKey() == true) {
+                return true
+            }
         }
+
         if (event.action == KeyEvent.ACTION_DOWN && event.keyCode == KeyEvent.KEYCODE_DEL) {
             // Check if the external lister has consumed the backspace pressed event
```

Comment:
minor typo, not strictly related to the change but let's fix this as we're on it: let's change `lister` for `listener`